client/dive-common/components/TrackDetailsPanel.vue

```diff
@@ -229,21 +240,64 @@ export default defineComponent({
           {{ type }}
         </option>
       </datalist>
-      <track-item
-        :solo="true"
-        :track="selectedTrack"
-        :track-type="selectedTrack.confidencePairs[0][0]"
-        :selected="true"
-        :editing="!!editingModeRef"
-        :input-value="true"
-        :color="typeStylingRef.color(selectedTrack.confidencePairs[0][0])"
-        :lock-types="lockTypes"
-        @seek="$emit('track-seek', $event)"
-      />
-
+      <v-card
+        v-for="track in selectedTrackList"
+        :key="track.trackId"
+        class="mx-2 mb-2"
+        outlined
+        flat
+      >
+        <track-item
+          :solo="true"
+          :merging="mergeInProgress"
+          :track="track"
+          :track-type="track.confidencePairs[0][0]"
+          :selected="true"
+          :editing="!!editingModeRef"
+          :input-value="true"
+          :color="typeStylingRef.color(track.confidencePairs[0][0])"
+          :lock-types="lockTypes"
+          @seek="$emit('track-seek', $event)"
+        />
+      </v-card>
+      <div class="d-flex flex-row">
+        <v-btn
+          :color="mergeInProgress ? 'error' : 'primary'"
+          class="mx-2 mb-2 grow"
+          depressed
+          x-small
+          @click="$emit('toggle-merge')"
+        >
+          <span v-if="!mergeInProgress">
+            <v-icon
+              small
+              class="pr-1"
+            >
+              mdi-call-merge
+            </v-icon>
+            Begin Track Merge (m)
+          </span>
+          <span v-else>
+            Abort (esc)
+          </span>
+        </v-btn>
+        <v-btn
+          v-if="mergeList.length >= 2"
+          color="success"
+          x-small
+          depressed
+          class="mr-2 mb-2 grow"
+          @click="$emit('commit-merge')"
+        >
+          <v-icon class="pr-1">
+            mdi-check
+          </v-icon>
+          commit (shift+m)
+        </v-btn>
+      </div>
       <confidence-subsection
         style="max-height:33vh"
-        :confidence-pairs="selectedTrack.confidencePairs"
+        :confidence-pairs="flatten(selectedTrackList.map((t) => t.confidencePairs))"
```

Comment:
I don't think it will be possible to make that make sense.  Even the relatively simple confidence pair list is a stretch. I've disabled the attributes in merge mode because all track editing should mostly be disabled during merge.